contrib/ohi-release-notes/action.yml

```
@@ -30,19 +30,19 @@ runs:
   using: composite
   steps:
     - name: Validate that the markdown is correct
-      uses: ./validate-markdown
+      uses: newrelic/release-toolkit/validate-markdown@v1
```

I am open to different solutions if available, I did a "quick fix" to unblock myself on the kafka PR

---

Oh! Good point, I did not think about that!

There are environment variables that are available only inside an action and there is one that is handy in composite actions: `GITHUB_ACTION_PATH` ([reference](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables))

Can you test this?

```suggestion
      uses: ${{ env.GITHUB_ACTION_PATH }}/validate-markdown
```

---

Hello! I tested it getting the error:
```
Unrecognized named-value: 'env'. Located at position 1 within expression: env.GITHUB_ACTION_PATH
```

I believe that the `env` is not available in the `uses` context 
 - https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability

Docs are not super clear about that though (`uses` keyword is not specified and neither composite actions  )

---

I am currently checking if we can make something static similar to the GITHUB_ACTION_PATH value.

Otherwise the only way I see to avoid having @v1 is to move this to a script 😖

---

After talking offline with @kang-makes we decided to keep it like this to avoid to implement a new script